software/samba/en.md

```diff
@@ -24,7 +24,7 @@ Please note that Samba does NOT run on system boot by default. It needs to be ma
 # Start Samba manually
 sudo systemctl start nmb smb
 
-# Configure Samba to start automatically on each boot
+# Configure Samba to start automatically on each boot and start it now
```

Comment:
IMO `start it now` should be changed to `immediately start the service`. It's less concise but slightly clearer.